cora/util/cubicspline.pyx

```diff
@@ -111,22 +110,22 @@ cdef class Interpolater(object):
         return self.value(x)
 
     @cython.boundscheck(False)
-    def value_array(self, x):
+    def value_array(self, double[::1]x):
```

Comment:
I think this change means that this routine will now only work on 1D arrays. Please add a test for this and fix if required. Removing the typing here and changing to something like `cdef double[::1] xr = np.ascontiguousarray(x.ravel())` should be sufficient.

Reply:
You are right that as it is, that limits usage to 1D arrays. 

Fun thing I learned, is that you can set the desire for a contiguous array in ravel: https://numpy.org/doc/stable/reference/generated/numpy.ravel.html.

However, I am still getting errors that it expects a 1D array, and am struggling to understand why....